src/main/fulcro/client/data_fetch.cljc

```diff
@@ -296,13 +302,19 @@
   be available for post mutations and fallbacks.
   "
   [env-or-app-state component-class ident field & params]
-  (let [params    (if (map? (first params)) (first params) params)
+  (let [params              (if (map? (first params)) (first params) params)
+        env?                (and (map? env-or-app-state) (contains? env-or-app-state :state))
+        env                 (if env?
+                              env-or-app-state
+                              {:state env-or-app-state})
+        load-marker-default (if env?
+                              (-> env :reconciler p/get-load-marker-default)
+                              true)
         {:keys [without params remote post-mutation post-mutation-params fallback parallel refresh marker abort-id]
-         :or   {remote :remote refresh [] marker true}} params
-        env       (if (and (map? env-or-app-state) (contains? env-or-app-state :state))
-                    env-or-app-state
-                    {:state env-or-app-state})
-        state-map (some-> env :state deref)]
+         :or   {remote :remote refresh [] marker load-marker-default}} params
+        state-map           (some-> env :state deref)]
+    (when-not env?
+      (log/warn "Warn about the fact that state means no way to get the default ?")) ; TODO remember to update
```

Comment:
This message should be 
```
(log/warn "load-field-action for field " field "was called with app state instead of env.  This is a deprecated usage and some features may not work when using it.")
```